jail-docs.yml

```diff
@@ -36,7 +40,7 @@
     target_commands:
     # `command` module does not allow to set env variables, nor expands them,
     # so let's use make's variables.
-    - "make -C master/docs PATH:={{ getent_passwd[worker_account].4 }}/{{ env_name }}/bin:$(PATH) VERSION=latest"
+    - "gmake -C master/docs PATH:={{ getent_passwd[worker_account].4 }}/{{ env_name }}/bin:$(PATH) VERSION=latest"
```

Comment:
Oh, interesting, gmake was installed already on bslave1 so I assumed it would be here, too.  It's not!